dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -161,3 +161,15 @@ OPENAPI_INFO_INVALID_SERVERXML_CWWKO1683W.useraction=Ensure that the title and v
 OPENAPI_MERGE_UNUSED_INCLUDE_SERVERXML_CWWKO1684W=CWWKO1684W: The {0} configuration element includes "{1}" but that does not match any deployed application or web module.
 OPENAPI_MERGE_UNUSED_INCLUDE_SERVERXML_CWWKO1684W.explanation=The server is configured to create OpenAPI documentation for the application or module name, but no application or web module with that name was deployed.
 OPENAPI_MERGE_UNUSED_INCLUDE_SERVERXML_CWWKO1684W.useraction=Check that all applications started correctly and that the names in the configuration are specified correctly.
+
+OPENAPI_MP_CONFIG_REDUNDANT_CWWKO1685I=CWWKO1685I: The {0} MicroProfile Config property is ignored because equivalent configuration is provided in server.xml.
+OPENAPI_MP_CONFIG_REDUNDANT_CWWKO1685I.explanation=When this property is set and the same function is configured using server.xml, the configuration in server.xml is used and the MicroProfile Config property is ignored.
+OPENAPI_MP_CONFIG_REDUNDANT_CWWKO1685I.useraction=Remove the MicroProfile Config property.
+
+OPENAPI_MP_CONFIG_CONFLICTS_CWWKO1686W=CWWKO1686W: The {0} MicroProfile Config property is ignored because conflicting configuration is provided in server.xml.
```

Comment:
```suggestion
OPENAPI_MP_CONFIG_CONFLICTS_CWWKO1686W=CWWKO1686W: The {0} MicroProfile Config property is ignored because conflicting configuration is provided in the server.xml file.
```